### PR TITLE
Update vault mixin

### PR DESCRIPTION
**Pull request by v-zhuravlev**
- Change panels to timeseries
- Change old bar charts to new bar gauges
- Add job, instance selectors
- Fix secrets table, add gauge to the table
- Add README

vaultscreen

### CODE REVIEW

File: vault-mixin/dashboards/vault.json [hunk 1]
```diff
@@ -2204,340 +2361,419 @@
...
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "${datasource}"
+        },
+        "definition": "label_values(vault_secret_kv_count{job=~\"$job\",instance=~\"$instance\"}, mount_point)",
         "hide": 0,
         "includeAll": true,
-        "label": "Mount Point:",
+        "label": "",
```
Comment (Dasomeone) [inline vault-mixin/dashboards/vault.json:2756]: If we have labels for `job` and `instance` it makes sense to also have it for the custom variables

**Review by Dasomeone — COMMENTED** [review]
Generally looks quite good, I like the additional panels!

I wonder if we could add a bit of context to the readme so there's a clear distinction between the "vault-mixin" and "vault" folders, though. Thoughts?

**Comment by v-zhuravlev** [comment]
> Generally looks quite good, I like the additional panels!
> 
> I wonder if we could add a bit of context to the readme so there's a clear distinction between the "vault-mixin" and "vault" folders, though. Thoughts?

Well, I think README here self describes what this mixin does?

File: vault-mixin/dashboards/vault.json (hunk 1 above)
Comment (v-zhuravlev) [reply vault-mixin/dashboards/vault.json]: updated.

**Comment by Dasomeone** [comment]
> > Generally looks quite good, I like the additional panels!
> > I wonder if we could add a bit of context to the readme so there's a clear distinction between the "vault-mixin" and "vault" folders, though. Thoughts?
> 
> Well, I think README here self describes what this mixin does?

Good point, I guess it's more a case of the other `vault` folder not being properly documented :0

**Pull request merged**